quizbizz/src/components/navbar/Navbar.jsx

```diff
@@ -17,9 +23,13 @@ const Navbar = () => {
       </div>
       <div className="nav-items-container-end">
         <div className="login-search-bar-container">
-          <Link to="/login">
-            <PrimaryButton buttonText="Login" />
-          </Link>
+          {!user.isUserLoggedIn ? (
+            <Link to="/login">
```

Comment:
suggestion: You can use nav link.

Reply:
I think it won't effect my functionality much because i dont want to show active state here